quartz/components/scripts/popover.inline.ts

```diff
@@ -91,7 +95,7 @@ async function mouseEnterHandler(
       normalizeRelativeURLs(html, targetUrl)
       // prepend all IDs inside popovers to prevent duplicates
       html.querySelectorAll("[id]").forEach((el) => {
-        const targetID = `popover-internal-${el.id}`
+        const targetID = `popover-${el.id}`
```

Comment:
this is different from the popover id, this is the id of the header inside the popover, its supposed to be mismatched from the actual popover id